canvas_sdk/effects/note/message.py

```diff
@@ -0,0 +1,107 @@
+import json
+from typing import Any
+from uuid import UUID
+
+from pydantic_core import InitErrorDetails
+
+from canvas_generated.messages.effects_pb2 import Effect
+from canvas_sdk.effects.note.base import NoteOrAppointmentABC
+from canvas_sdk.v1.data import Message as MessageModel
+from canvas_sdk.v1.data import Patient, Staff
+
+
+class Message(NoteOrAppointmentABC):
```

Comment:
Why would we inherit from `NoteOrAppointmentABC` ? We dont have the same error details with the provider or practice location

Reply:
No, but it's conceptually part of the Note/Appointment system, so I've inherited from the ABC that gives me the `TrackableFieldsModel`, the `create` method. It's not at all needed and I could've duplicated a few things instead

Reply:
I've refactored this to actually not be a part of a `NoteOrAppointmentABC` 👍